clients/java/client/src/it/java/org/operaton/bpm/client/rule/EngineRule.java

```diff
@@ -283,11 +283,14 @@ else if (typedValue instanceof FileValueImpl) {
       payload.append("{");
       if (!map.isEmpty()) {
         String variablesAsString = objectMapper.writeValueAsString(map);
-        payload.append("\"variables\": " + variablesAsString);
-        payload.append(",");
+        payload.append("\"variables\": ")
+                .append(variablesAsString)
+                .append(",");
       }
       if (businessKey != null) {
-        payload.append("\"businessKey\": \"" + businessKey + "\"");
+          payload.append("\"businessKey\": \"")
```

Comment:
this changeset contains several indentation changes. The default is 2 spaces. Please revise the changes and change indentation.